.sbt_metadata/reindex_worker.json

```diff
@@ -2,6 +2,7 @@
   "id" : "reindex_worker",
   "folder" : "reindexer/reindex_worker",
   "dependencyIds" : [
+    "internal_model",
```

Comment:
could we create a new module/library instead of shoving the adpater/transformer models into the `internal model`?
I don't think the adapters/reindexer need to know about the `Work` model and the other stages of the pipeline don't need to know about the source data. I think it's best to keep them separate

Reply:
Does that mean the adapter/reindexer services could also be in a separate repo?

Reply:
I don’t think so. The point of this work is to unify the transformers, adapters, and reindexer. If anything, it makes it harder to split two of them out.